src/style.css

```diff
@@ -0,0 +1,132 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  padding-top: 30px;
+  background-color: rgb(242 242 242);
+}
+
+main {
+  width: 50%;
+  margin: 0 auto;
+  background-color: white;
+  box-shadow: 5px 10px 18px #9e9e9e;
+}
+
+.toDoItem {
+  display: flex;
+  flex-direction: row;
+  padding: 12px;
+  border-bottom: 1px solid #e4e4e4;
+  gap: 12px;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.hidden {
+  display: none;
+}
+
+.deleter {
+  cursor: pointer;
+  color: #a5a3a3;
+}
+
+.dragger {
+  cursor: move;
+  color: #b9b9b9;
```

Comment:
Kindly as a suggestion try to use CSS variables to help you with a more flexible and scalable code over time.